HelpSource/Classes/VolumeGui.schelp

```diff
@@ -0,0 +1,27 @@
+CLASS:: VolumeGui
+SUMMARY:: Acts as a Frontend to Volume Class
+CATEGORIES:: Server
+RELATED:: Classes/Server, Classes/Server/Volume
+
+DESCRIPTION::
+VolumeGui is an abstraction method used by Volume to create a graphic user interface. If one looks into the volume source file, and look into the gui method, one may find the VolumeGui abstraction being used to instantiate the Volume UI Frontend.
+
+CLASSMETHODS::
+method:: new
+Creates a new window for VolumeGui new window when instanced
+
+CODE::
+v = s.volume;
```

Comment:
@tiagmoraismorgado Again, about the code example.. Maybe don't add examples for abstract classes? Or as @jamshark70 said, ask on the list first before submitting the PR?